### ISSUE TITLE
SDK forces NFC (android.hardware.nfc) to be required feature

### ISSUE THREAD

**Issue by TatsuUkraine** [stripe/stripe-terminal-android#420]
## Summary
After upgrading to 3.2 install base reduced tremendously. It seems in most recent update `android.hardware.nfc` become required feature to be available on a device

## Code to reproduce
Use example project with localmobile SDK

## Android version
Any

## Impacted devices (Android devices or readers)
Devices without NFC reader

## SDK version
3.2.1

## Other information
In prev releases NFC wasn't required feature, but in most recent manifest has this declaration
```
    <uses-feature
        android:name="android.hardware.nfc"
        android:required="true" />
```

**Comment by chr-stripe** [comment]
Hi @TatsuUkraine, thank you for bringing this to our attention. This was an unintentional change and we will fix this in a future localmobile SDK release.

In the meantime, I believe you should be able to strip this requirement so that you can continue to install your application on devices that do not support NFC. Please confirm that you can add the following snippet to your application's `AndroidManifest.xml` file in order to temporarily remove this new addition:

```xml
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools">

    <uses-feature
        android:name="android.hardware.nfc"
        android:required="false"
        tools:replace="android:required" />

    <!-- the rest of your AndroidManifest.xml file -->

</manifest>
```

This should remove NFC as a requirement at install/upgrade time for now. I will update this ticket once we have shipped a version of the SDK that contains a fix and no longer requires a manual workaround.

**Comment by TatsuUkraine** [comment]
Hi @chr-stripe , thank you, yes, this workaround is what I wanted to patch asap in our project)

**Comment by chr-stripe** [comment]
This is now fixed in SDK 3.4.0; the `android.hardware.nfc` feature is no longer marked as required.

**Comment by TatsuUkraine** [comment]
Thanks!
